controllers/usersignup/usersignup_controller.go

```diff
@@ -548,7 +548,7 @@ func (r *Reconciler) DeleteMasterUserRecord(mur *toolchainv1alpha1.MasterUserRec
 
 	err := r.updateStatus(logger, userSignup, inProgressStatusUpdater)
 	if err != nil {
-		return nil
+		return err
```

Comment:
I guess we are missing a test for this case? Otherwise we would catch it earlier.

Reply:
ah right, I intended to add a test for that, as I did in https://github.com/codeready-toolchain/member-operator/pull/328/files#diff-f57f1b01b263fd00b64ab1ec2d007285e03675feb420c7732360f639b38ce736R1262 when making a similar change, but I forgot about it 🤦‍♂️ ! Thanks for the reminder ;)

Reply:
done in 1af071a by adding a test inspired from `TestUserSignupDeactivatedButMURDeleteFails` just above